experimental/index.html

```diff
@@ -100,21 +108,269 @@
         datacommons.drawLine(lineChartElement2, lineChartProps2);
       };
     </script>
-    <link rel="stylesheet" href="../server/dist/css/ranking.min.css" />
   </head>
-  <body>
-    <div>
-      <h1>Render Data Commons charts from a static HTML page</h1>
-    </div>
-    <div class="flex-container flex-column">
-      <div class="flex-container flex-row">
-        <div id="my-line-chart-1" class="flex-child"></div>
-        <div id="my-line-chart-2" class="flex-child"></div>
-      </div>
-      <div class="flex-container flex-row">
-        <div id="my-component-1" class="flex-child"></div>
-        <div id="my-component-2" class="flex-child"></div>
+  <body class="flex flex-column">
+    <nav
+      class="flex items-center justify-between flex-wrap border-b-2 border-slate-200 px-4 py-3"
+    >
+      <div class="flex items-center mx-auto dc-content">
+        <div class="flex items-center flex-shrink-0 text-slate-900 mr-6">
+          <span class="font-light text-xl tracking-tight"
+            >Sustainable Development Goals Tracker</span
+          >
+        </div>
+        <div class="block lg:hidden">
+          <button
+            class="flex items-center px-3 py-2 border rounded text-blue-600 border-teal-400 hover:text-slate-500 hover:border-white"
+          >
+            <svg
+              class="fill-current h-3 w-3"
+              viewBox="0 0 20 20"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <title>Menu</title>
+              <path d="M0 3h20v2H0V3zm0 6h20v2H0V9zm0 6h20v2H0v-2z" />
+            </svg>
+          </button>
+        </div>
+
+        <div class="w-full block flex-grow lg:flex lg:items-center lg:w-auto">
+          <div class="text-sm lg:flex-grow">
+            <a
+              href="#responsive-header"
+              class="block mt-4 lg:inline-block lg:!mt-0 text-sky-400 hover:text-sky-600 hover:no-underline mr-4"
+            >
+              About
+            </a>
+          </div>
+          <div>
+            <div class="relative text-gray-600">
+              <input
+                type="search"
+                name="serch"
+                placeholder="Country"
+                class="bg-white h-10 px-5 pr-10 rounded-full text-sm focus:outline-none shadow-sm"
+              />
+              <button type="submit" class="absolute right-0 top-0 mt-3 mr-4">
+                <svg
+                  class="h-4 w-4 fill-current"
+                  xmlns="http://www.w3.org/2000/svg"
+                  xmlns:xlink="http://www.w3.org/1999/xlink"
+                  version="1.1"
+                  id="Capa_1"
+                  x="0px"
+                  y="0px"
+                  viewBox="0 0 56.966 56.966"
+                  style="enable-background: new 0 0 56.966 56.966"
+                  xml:space="preserve"
+                  width="512px"
+                  height="512px"
+                >
+                  <path
```

Comment:
wonder for button and other UI, can use bootstrap, jquery or other popular components? Then the sdg name could be wrapped in the javascript as an array.

Reply:
that sounds reasonable. Added jquery, lodash, and bootstrap, and removed tailwind